compose/migrate.md

```diff
@@ -63,51 +77,43 @@ The following behave differently between Compose V1 and V2:
 
 ### Environment variables
 
-Environment variable behavior in Compose V1 was not formally documented and behaved inconsistently in some edge cases.
+Environment variable behavior in Compose V1 wasn't formally documented and behaved inconsistently in some edge cases.
 
-For Compose V2, the [Environment variables](/compose/environment-variables/) section covers both [precedence](/compose/environment-variables/envvars-precedence) as well as [`.env` file interpolation](/compose/environment-variables/env-file) and includes numerous examples covering tricky situations such as escaping nested quotes.
+For Compose V2, the [Environment variables](/compose/environment-variables/) section covers both [precedence](/compose/environment-variables/envvars-precedence) as well as [`.env` file interpolation](/compose/environment-variables/env-file) and includes many examples covering tricky situations such as escaping nested quotes.
 
-Ask yourself...
-- Is my project using multiple levels of environment variable overrides (e.g. `.env` file and `--env` CLI flags)?
-- Do any `.env` file values have escape sequences or nested quotes?
-- Do any `.env` file values contain literal `$` signs in them? (This is common with PHP projects!)
-- Do any variable values use advanced expansion syntax? (e.g. `${VAR:?error}`)
+Check if:
+- Your project uses multiple levels of environment variable overrides, for example `.env` file and `--env` CLI flags
+- Any `.env` file values have escape sequences or nested quotes.
+- Any `.env` file values contain literal `$` signs in them. This is common with PHP projects.
+- Any variable values use advanced expansion syntax, for example `${VAR:?error}`.
 
-Run `docker compose config` on the project to preview the configuration after Compose V2 has performed interpolation to
+> **Tip**
+>
+> Run `docker compose config` on the project to preview the configuration after Compose V2 has performed interpolation to
 verify that values appear as expected.
-
-Maintaining backwards compatibility with Compose V1 is typically achievable by ensuring that literal values (no
+>
+> Maintaining backwards compatibility with Compose V1 is typically achievable by ensuring that literal values (no
 interpolation) are single-quoted and values that should have interpolation applied are double-quoted.
+{: .tip}
 
 ## What does this mean for my projects that use Compose V1?
 
 For most projects, switching to Compose V2 requires no changes to the Compose YAML or your development workflow.
 
-We recommend that you adapt to the new preferred way of running Compose V2, which is to use `docker compose` instead of `docker-compose`.
+It's recommend that you adapt to the new preferred way of running Compose V2, which is to use `docker compose` instead of `docker-compose`.
```

Comment:
Tiny typo:
```suggestion
It's recommended that you adapt to the new preferred way of running Compose V2, which is to use `docker compose` instead of `docker-compose`.
```